addons/website_slides/models/slide_question.py

```diff
@@ -15,6 +15,8 @@ class SlideQuestion(models.Model):
     question = fields.Char("Question Name", required=True, translate=True)
     slide_id = fields.Many2one('slide.slide', string="Content", required=True, ondelete='cascade')
     answer_ids = fields.One2many('slide.answer', 'question_id', string="Answer", copy=True)
+    correct_answers = fields.Integer(compute='_compute_answers')
```

Comment:
We usually use an explicit `_count` suffix for this kind of fields. Also, if we need to know if there are multiple correct ones, why not use a boolean field? 
It will also be directly usable in your templates without the need to add to the controller.